src/app/components/dashboard-apps-dock/dashboard-apps-dock.coffee

```diff
@@ -10,6 +10,7 @@ DashboardAppsDockCtrl = ($scope, $cookies, $uibModal, $window, $timeout, MnoeOrg
   $scope.popoverTemplateUrl = 'app/components/dashboard-apps-dock/no-apps-notification.html'
   $scope.isMarketplaceEnabled = MARKETPLACE_CONFIG.enabled
   $scope.isOnboardingEnabled = ONBOARDING_WIZARD_CONFIG.enabled
+  $scope.isPopoverShown = $scope.isOnboardingEnabled && $scope.isMarketplaceEnabled && !$scope.isLoading && _.isEmpty($scope.apps)
```

Comment:
This should stay a function, `!$scope.isLoading` will only be evaluated at init time.

Reply:
Sorry @alexnoox , I went to add the fix blindly. In that case, can you please have a look at the new commit?